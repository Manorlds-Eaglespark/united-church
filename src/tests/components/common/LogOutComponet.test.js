import React from 'react';
import { shallow } from 'enzyme';
import LogOutComponent from '../../../components/common/LogOutComponent';


describe('<LogOutComponent />', () => {
  it('should match snapshot', () => {
    const props = {
      handleLogOut: jest.fn(),
    };
    const wrapper = shallow(<LogOutComponent {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
