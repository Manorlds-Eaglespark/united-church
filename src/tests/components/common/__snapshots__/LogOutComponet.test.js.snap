// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LogOutComponent /> should match snapshot 1`] = `ShallowWrapper {}`;
